src/writing-your-own-application/generate-project/esp-idf-template.md

```diff
@@ -91,7 +91,7 @@ cargo run
 
 This builds the code according to the configuration and executes [`espflash`][espflash]  to flash the code to the board.
 
-Since our [`runner` configuration][runner-config] also passes the `--monitor` argument to [`espflash`][espflash]  we can see what the code is printing.
+Since our [`runner` configuration][runner-config] also passes the `--monitor` argument to [`espflash`][espflash] we can see what the code is printing.
```

Comment:
Looks like all these in-line code snippets disoriented Grammarly. This is the complex sentence with a dependent clause followed by an independent clause. That's why you need a comma before "we can see".

... passes the `--monitor` argument to [`espflash`][espflash], we can see ...